devices/Bq2579x/Bq2579x.cs

```diff
@@ -261,6 +268,58 @@ public class Bq2579x : IDisposable
         [Property]
         public ThermalShutdownThreshold ThermalShutdownThreshold { get => GetThermalShutdownThreshold(); set => SetThermalShutdownThreshold(value); }
 
+        /// <summary>
+        /// Gets or sets charge voltage for JEITA high temperature range (TWARN � THOT). 
+        /// </summary>
+        [Property]
+        public ChargeVoltage ChargeVoltageHighTempRange { get => GetChargeVoltage(); set => SetChargeVoltage(value); }
+
+        /// <summary>
+        /// Gets or sets charge current for JEITA high temperature range (TWARN � THOT). 
+        /// </summary>
+        [Property]
+        public ChargeCurrent ChargeCurrentHighTempRange { get => GetChargeCurrentHighTempRange(); set => SetChargeCurrentHighTempRange(value); }
+
+        /// <summary>
+        /// Gets or sets charge current for JEITA low temperature range (TCOLD � TCOOL). 
+        /// </summary>
+        [Property]
+        public ChargeCurrent ChargeCurrentLowTempRange { get => GetChargeCurrentLowTempRange(); set => SetChargeCurrentLowTempRange(value); }
+
+        /// <summary>
+        /// Gets or sets VT2 comparator voltage rising thresholds as a percentage of REGN.
+        /// </summary>
+        [Property]
+        public CompVoltageRisingThreshold Vt2ComparatorRisingThreshold { get => GetVt2ComparatorVoltage(); set => SetVt2ComparatorVoltage(value); }
+
+        /// <summary>
+        /// Gets or sets VT3 comparator voltage falling thresholds as a percentage of REGN.
+        /// </summary>
+        [Property]
+        public CompVoltageFallingThreshold Vt3ComparatorFallingThreshold { get => GetVt3ComparatorVoltage(); set => SetVt3ComparatorVoltage(value); }
+
+        /// <summary>
+        /// Gets or sets OTG mode TS HOT temperature threshold.
```

Comment:
a quick definition of OTG may help as well

Reply:
same here...